src/test/java/marquez/service/mappers/DataSourceRowMapperTest.java

```diff
@@ -1,57 +1,46 @@
 package marquez.service.mappers;
 
-import static marquez.common.models.Description.NO_DESCRIPTION;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 
-import java.util.Optional;
+import marquez.UnitTests;
 import marquez.common.models.ConnectionUrl;
 import marquez.common.models.DataSource;
+import marquez.common.models.DbName;
 import marquez.common.models.DbSchemaName;
 import marquez.common.models.DbTableName;
-import marquez.common.models.Description;
 import marquez.db.models.DataSourceRow;
 import marquez.service.models.DbTableVersion;
 import org.junit.Test;
+import org.junit.experimental.categories.Category;
 
+@Category(UnitTests.class)
 public class DataSourceRowMapperTest {
   private static final DataSource DATA_SOURCE = DataSource.fromString("postgresql");
+  private static final DbName DB_NAME = DbName.fromString("test_db");
   private static final ConnectionUrl CONNECTION_URL =
       ConnectionUrl.fromString(
-          String.format("jdbc:%s://localhost:5432/novelists", DATA_SOURCE.getValue()));
-  private static final DbSchemaName DB_SCHEMA_NAME = DbSchemaName.fromString("marquez");
-  private static final DbTableName DB_TABLE_NAME = DbTableName.fromString("quotes");
-  private static final Description DESCRIPTION =
-      Description.fromString("It's enough for me to be sure that you and I exist as this moment.");
+          String.format("jdbc:%s://localhost:5432/%s", DATA_SOURCE.getValue(), DB_NAME.getValue()));
```

Comment:
Are you sure we want to change the description here? Seems like the current value more closely resembles what it would be than the new value?

Reply:
The diff is hard to read, but the description isn't changed, but rather removed. We are setting this value to `CONNECTION_URL`. I'm going to review test to make sure I didn't remove `DESCRIPTION` unintentionally (which might be the case, actually).

Reply:
The description was removed because a `DataSourceRow` does not have a description.

Reply:
Ah, right. Sorry about that!